apps/backend/api/src/routes/question.route.ts

```diff
@@ -47,7 +47,7 @@ questionRouter.delete(
 
 questionRouter.get(
   '/',
-  validateSplit(undefined, undefined, Schema.QuestionWhereInputSchema),
+  validateSplit(Schema.QuestionWhereInputSchema, undefined, undefined),
   asyncHandler(getAllQuestionHandler)
 );
```

Comment:
it's strange because all my postman request works with this formulation

Reply:
Well, maybe I'm wrong? We should ask @Apoorva64

Reply:
Should be using query ( functions takes in (
  paramsSchema?: ZodType,
  querySchema?: ZodType,
  bodySchema?: ZodType
)
So should be validateSplit(undefined, Schema.QuestionWhereInputSchema, undefined)